### PR TITLE
Update tests to use Logstash 5.0.0-alpha1

### PR DESCRIPTION
**Pull request by andrewkroh**

### CODE REVIEW

File: testing/environments/docker/logstash/Dockerfile-5.0.0-alpha1 [hunk 1]
```diff
@@ -3,7 +3,7 @@ FROM java:8-jre
 ENV LS_VERSION 2
```
Comment (urso) [inline testing/environments/docker/logstash/Dockerfile-5.0.0-alpha1]: this still used?

Comment (andrewkroh) [inline testing/environments/docker/logstash/Dockerfile-5.0.0-alpha1]: Not sure what that was used for. I'll set it to `5`.

Comment (andrewkroh) [inline testing/environments/docker/logstash/Dockerfile-5.0.0-alpha1]: It is used to handle config file changes between 1.5 and 2+.

**Comment by tsg** [comment]
LGTM.

**Pull request merged**